openqabot/utils.py

```diff
@@ -80,10 +81,14 @@ def merge_dicts(dict1: dict[Any, Any], dict2: dict[Any, Any]) -> dict[Any, Any]:
     return copy
 
 
+def number_of_retries(fallback: int) -> int:
+    return 0 if "PYTEST_VERSION" in os.environ else int(os.environ.get("QEM_BOT_RETRIES", fallback))
```

Comment:
How about specifying the default already here

```suggestion
def number_of_retries(fallback: int = 3) -> int:
    return 0 if "PYTEST_VERSION" in os.environ else int(os.environ.get("QEM_BOT_RETRIES", fallback))
```